### DISCUSSION TITLE
certificate expired for self hosted database

### DISCUSSION THREAD

**Discussion by islamyakin** [gravitational/teleport#36170] [Q&A] [answered]
hello everyone, I'm a teleport user.
I have a problem with the SSL expiration time used by the database, and generated by Teleport. in the teleport proxy server I use ACME SSL.
My question is, can each SSL generated by Teleport have its expiry time set to 1 year?
```sudo tctl auth sign --format=db --host=localhost --out=db --ttl=2190h```
However, after 3 months, the SSL can no longer be used. is there a solution to my problem.

example cert gen by teleport
![image](https://github.com/gravitational/teleport/assets/73155834/1338337b-6561-444b-bd3f-107c4b26bbb2)

thank u

**Comment by stevenGravy** [comment, marked as answer]
Currently `2190` hours would be around 3 months so you can increase that.  Whatever you set it as I would have a reminder so you can make sure to update it.

**Reply by webvictim** [reply]
Yes - `--ttl 8760h` should give you a certificate valid for a year.
